tests/integration/components/supplementary-file-browser-test.js

```diff
@@ -67,16 +112,16 @@ test('has additional files', function(assert) {
     assert.equal(this.$('#rightArrow').length, 1);
     assert.equal(this.$('#downArrow').length, 1);
 
+    // TODO enable when when tests are fixed
```

Comment:
Makes sense. For suggestions, the only thing I can think of is pinging @jamescdavis, as he was telling me about a `wait` method that could be helpful here. However, I am a little worried about this not being testable and causing other things to break and suggest that maybe this gets looked into/fixed prior to it going to code review and getting merged.